bench/string-searching.js

```diff
@@ -0,0 +1,29 @@
+const Benchmark = require('benchmark')
+const { eventToMdTable, H2, createTableHeader } = require('../markdown')
+const suite = new Benchmark.Suite;
+
+const tableHeader = createTableHeader([
+  'name',
+  'ops/sec',
+  'samples'
+])
+
+const text = 'text/html,application/xhtml+xml,application/xml;application/json;q=0.9,image/avif,image/webp,*/*;q=0.8'
+
+suite.add('Using includes', function () {
+  text.includes('application/json')
+})
+.add('Using indexof', function () {
+  text.indexOf('application/json') !== -1
+})
+.add('Using RegExp.test', function () {
+  /application\/json/.test(text)
+})
```

Comment:
I think you can include another one with RegExp cached.

Reply:
You've lost me @H4ad, mind suggesting an example?

Reply:
```js
const testJson = /application\/json/;

.add('Usingcached RegExp.test', function () {
  testJson.test(text)
})
```

Reply:
Added!